### PR TITLE
wip: add main visual design page

### LINKED ISSUE #1: basic design coding

### PR DESCRIPTION
**Pull request by yue4u**
close #1

### CODE REVIEW

File: frontend/components/design/Main.vue [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <main-visual />
+  <div class="bg-yellow-900">
+    <div class="container mx-auto">
+      <Sentence />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
```
Comment (yue4u) [inline frontend/components/design/Main.vue]: please use `<script setup lang="ts"/>`, you can find  links in https://github.com/rainy-me/tsukiyo/blob/d351653bc83e1ed89ed56106d8c02c96f945d1db/frontend/admin/EventEditor.vue#L37

File: frontend/components/design/MainVisual.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    >
+      <div class="w-24">
+        <div class="text-8xl leading-tight">月夜のけだもの</div>
+      </div>
+    </div>
+    <Nav />
+  </div>
+</template>
+
+<script lang="ts">
```
Comment (yue4u) [inline frontend/components/design/MainVisual.vue]: same as above

File: frontend/components/design/Nav.vue [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  <div class="absolute text-xl top-0 right-0 h-full flex items-center mr-10">
+    <ul>
+      <li v-for="item in menu" :key="item.text" class="my-8">
+        <a :href="item.link">{{ item.text }}</a>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts">
```
Comment (yue4u) [inline frontend/components/design/Nav.vue]: same

File: frontend/components/design/Sentence.vue [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  <div class="flex max-h-screen py-20 justify-between">
+    <img class="w-3/5" src="/assets/DSC08504.jpeg" alt="" />
+    <div class="w-2/5 text-white text-center">
+      <div class="text-4xl my-10 inline-block under">宮沢賢治</div>
+      <div class="mx-10">{{ text }}</div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
```
Comment (yue4u) [inline frontend/components/design/Sentence.vue]: same(this is the last one) :1st_place_medal:

File: frontend/components/design/Sentence.vue [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  data() {
+    return {
+      text:
+        "紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文紹介文",
+    };
+  },
+};
+</script>
+
+<style>
```
Comment (yue4u) [inline frontend/components/design/Sentence.vue]: when writing a style rule in sfc, please use `scope` to encapsulate it

**Review by yue4u — COMMENTED** [review]
@koorimizuw 

> https://github.com/rainy-me/tsukiyo/pull/8#issuecomment-762480272

please reduce / adjust img sizes!

**Review by koorimizuw — COMMENTED** [review]
安拉天下第一

**Pull request merged**